docs/pages/reference/helm-reference/includes/zz_generated.teleport-kube-agent.mdx

```diff
@@ -0,0 +1,1627 @@
+
+## `roles`
+
+| Type | Default |
+|------|---------|
+| `string` | `"kube"` |
+
+`roles` is a comma-separated list of services which should be enabled
+when running the `teleport-kube-agent` chart.
+
+| Services                     | Value for `roles` | Mandatory additional settings for this role |
+|------------------------------|-------------------|---------------------------------------------|
+| Teleport Kubernetes service  | `kube`            | [`kubeClusterName`](#kubeclustername)       |
+| Teleport Application service | `app`             | [`apps`](#apps) or [`appResources`](#appresources) |
+| Teleport Database service    | `db`              | [`databases`](#databases) or [`databaseResources`](#databaseresources) |
+| Teleport Discovery service   | `discovery`       | [`kubeClusterName`](#kubeclustername) |
+
+For example: `roles: kube,app,discovery`.
```

Comment:
I would make this a YAML code snippet so it's more immediately clear what this means.